### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by jkobject**

### CODE REVIEW

File: ccle_tasks/generate_ccle_cellline_dataset.ipynb [hunk 1]
```diff
@@ -40,7 +40,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "track = tracker.getTracker()"
+    "track = tracker.getCCLETracker()"
```
Comment (jkobject) [inline ccle_tasks/generate_ccle_cellline_dataset.ipynb:43]:
```suggestion
    "track = tracker.getTracker()"
```

File: depmapomics/loading.py [hunk 2]
```diff
@@ -762,7 +767,7 @@ def load(samplesetname, workspaces,
   return samples
 
 
-def updateWES(samples, samplesetname, bucket=WES_GCS_PATH,
+def updateWES(samples, samplesetname, bucket="gs://cclebams/wes/",
```
Comment (jkobject) [inline depmapomics/loading.py:770]:
```suggestion
def updateWES(samples, samplesetname, bucket=WES_GCS_PATH,
```

File: depmapomics/tracker.py [hunk 3]
```diff
@@ -365,12 +403,13 @@ def resolveIssues(tracker, issus, arxspid, cols):
...
   """
   #for val in issus:
+  return False
 
 
 def retrieveFromCellLineName(noarxspan, ccle_refsamples, datatype, extract={}, my_id='~/.client_secret.json',
                             stripped_cell_line_name="stripped_cell_line_name", arxspan_id="arxspan_id",
                             mystorage_id="~/.storage.json",
-                            depmappvlink=DEPMAP_PV):
+                            depmappvlink="https://docs.google.com/spreadsheets/d/1uqCOos-T9EMQU7y2ZUw4Nm84opU5fIT1y7jet1vnScE"):
```
Comment (jkobject) [inline depmapomics/tracker.py:412]:
```suggestion
                            depmappvlink=DEPMAP_PV):
```

**Review by jkobject — COMMENTED** [review]
will need more reviews. need to add back the stjude_rms data

File: depmapomics/qc/test_compare_to_ref_release.py [hunk 4]
```diff
@@ -290,17 +290,3 @@ def test_mutation_legacy_data(data):
 #     data1_ = data1.loc[row, col].T
 #     data2_ = data2.loc[row, col].T
 #     assert_frame_equal(data1_, data2_, rtol=rtol)
-
-PARAMS_sharedcellines_are_equal = [((x['file'], x['merge_cols']), x['merge_cols'])
```
Comment (jkobject) [inline depmapomics/qc/test_compare_to_ref_release.py:294]: is this change ok? if so we can merge dev into master

File: depmapomics/copynumbers.py [hunk 5]
```diff
@@ -36,8 +37,25 @@ def loadFromGATKAggregation(refworkspace,  sortby=[SAMPLEID, 'Chromosome', "Star
...
+      colname (str, optional): the column in Terra where the file is saved in sampleset. Defaults to "combined_seg_file".
+      plotColname (str, optional): the column on terra where the plots exist in sample. Defaults to "modeled_segments_plot_tumor".
+      tempFolder (str, optional): where to put temp files. Defaults to "temp/".
+      toremove (list, optional): columns in Terra samples to remove (and delete corresponding data). Defaults to ["readgroup_ubams", ].
+      sampleset (str, optional): sample set to load in terra. Defaults to "all".
+      colRenaming (dict, optional): segment renaming dict. Defaults to COLRENAMING.
+
+  Returns:
+      pd.dataframe: dataframe containing the segments concatenated in a bed like format
+  """  """"""
```
Comment (jkobject) [inline depmapomics/copynumbers.py:58]:
```suggestion
  """
```

File: depmapomics/copynumbers.py [hunk 6]
```diff
@@ -347,10 +359,9 @@ def _CCLEPostProcessing(wesrefworkspace=WESCNWORKSPACE, wgsrefworkspace=WGSWORKS
 
   # annotating source
   for v in set(wgssegments[SAMPLEID]):
-    if len(tracker[tracker.index == v].source.values) > 0:
-      wgssegments.loc[wgssegments[wgssegments[SAMPLEID] == v].index,
+    wgssegments.loc[wgssegments[wgssegments[SAMPLEID] == v].index,
                         'Source'] = tracker[tracker.index == v].source.values[0]
-  wgssegments.Source = wgssegments.Source.replace(source_rename)
+    wgssegments.Source = wgssegments.Source.replace(source_rename)
```
Comment (jkobject) [inline depmapomics/copynumbers.py:364]:
```suggestion
  wgssegments.Source = wgssegments.Source.replace(source_rename)
```

File: depmapomics/copynumbers.py [hunk 7]
```diff
@@ -475,19 +484,38 @@ def _ProcessForAchilles(wespriosegs, wgspriosegs, samplesetname=SAMPLESETNAME, b
                         'ACH-001741',
                         'ACH-000010',
                         'ACH-002475',
-                        'ACH-001543'], taiga_legacy_loc=TAIGA_LEGACY_CN,
```
Comment (jkobject) [inline depmapomics/copynumbers.py:478]: need to remove all these

File: depmapomics/copynumbers.py [hunk 8]
```diff
@@ -475,19 +484,38 @@ def _ProcessForAchilles(wespriosegs, wgspriosegs, samplesetname=SAMPLESETNAME, b
...
+                        'ACH-001543'
+                        # TODO: drop these elements directly in the legacy datasets and should be blacklisted in the tracker
+                        ], taiga_legacy_loc='depmap-wes-cn-data--08f3',
```
Comment (jkobject) [inline depmapomics/copynumbers.py:489]:
```suggestion
                        ], taiga_legacy_loc= TAIGA_LEGACY_CN,
```

File: depmapomics/copynumbers.py [hunk 9]
```diff
@@ -475,19 +484,38 @@ def _ProcessForAchilles(wespriosegs, wgspriosegs, samplesetname=SAMPLESETNAME, b
...
                        taiga_legacy_filename='legacy_segments',
-                       taiga_dataset=TAIGA_CN_ACHILLES,
+                       taiga_dataset="cn-wes-achilles-4dcd",
```
Comment (jkobject) [inline depmapomics/copynumbers.py:491]:
```suggestion
                       taiga_dataset= TAIGA_CN_ACHILLES,
```

File: depmapomics/qc/test_compare_to_ref_release.py (hunk 4 above)
Comment (jkobject) [reply depmapomics/qc/test_compare_to_ref_release.py:294]: @javadnoorb

Comment (javadnoorb) [reply depmapomics/qc/test_compare_to_ref_release.py:294]: yes, I think I added a bunch of more specific tests that covered this. So I dropped it. Thanks for checking.

**Comment by javadnoorb** [comment]
@jkobject  Should we maybe wait until the release is done before merging dev into master? A few days ago when I checked I noticed that the dev/master switch didn't work, so master was using the sandbox tracker.

**Comment by jkobject** [comment]
@javadnoorb it is actually the opposite. dev contains many debugs of master. so we need to merge it before starting the release.

The goal was to test run dev beforehand but we didn't do it. Master was not test run either and contains more bugs than dev (dev = master + debugs)

**Pull request merged**